etc/conda-activate.sh

```diff
@@ -55,6 +59,7 @@ export CUSPATIAL_ROOT="$CUSPATIAL_HOME/cpp/build/$(basename "$CUSPATIAL_ROOT_ABS
 export CUDF_JNI_ROOT="$CUDF_HOME/java/src/main/native/build/$(basename "$CUDF_JNI_ROOT_ABS")"
 
 export RMM_LIBRARY="$RMM_ROOT/librmm.so"
+export KVIKIO_LIBRARY="$KVIKIO_ROOT/libkvikio.so"
```

Comment:
I thought this was header-only... same for `librmm.so`?

Reply:
Correct on both counts. I will clear those out.